applications/nrf5340_audio/broadcast_sink/main.c

```diff
@@ -303,6 +309,97 @@ static void le_audio_msg_sub_thread(void)
 	}
 }
 
+/**
+ * @brief	Handle bt_mgmt events.
+ */
+static void bt_mgmt_msg_sub_thread(void)
+{
+	int ret;
+	static uint8_t broadcast_code[BT_AUDIO_BROADCAST_CODE_SIZE];
```

Comment:
As we discussed, this should not be stored here. Should just send a pointer in to `bt_mgmt_broadcast_code_get` and pass that on to `broadcast_sink_broadcast_code_set`

Reply:
This will be fixed by OCT-3087, in a separate PR?